src/lib/Controls.js

```diff
@@ -214,7 +217,7 @@ class Controls {
         }
 
         cell.appendChild(element);
-        this.controlsEl.appendChild(cell);
+        parentElement.appendChild(cell);
```

Comment:
Can we repurpose `add` to just create the element and then defer the `appendChild` part to the calling method? It looks like we're already returning `element` as our result.

Reply:
It would probably end up looking something like the following in each `add` call:
 ```
const control = controls.add(...);
controls.controlsEl.appendChild(control);
```

It requires the calling code to know where to put it, but really only the `Controls` instance knows that. Thoughts?

Reply:
Yeah, we probably should avoid optimizing it too much considering we may want to rewrite these components eventually. Go ahead and disregard my initial comment.